Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+ENV WDIR=/data
+WORKDIR $WDIR
+
+# tag to use
+ENV WTAG=1.5.5
```

Comment:
How about automatically extracting the tag from here: https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/__init__.py#L9 ?

Reply:
ok, I can try, but I need to know if the tag is properly updated in master/main github branch. I can move this part down when I do `git clone` and then rebase to whatever tag will be provided by __init__.py file.